Detectors/TRD/simulation/src/Detector.cxx

```diff
@@ -126,35 +118,58 @@ bool Detector::ProcessHits(FairVolume* v)
   cIdSector[2] = 0;
   sector = atoi(cIdSector);
   // The detector number (0 – 539)
-  det = mGeom->getDetector(mGeom->getLayer(idChamber),
-                           mGeom->getStack(idChamber), sector);
+  det = mGeom->getDetector(mGeom->getLayer(idChamber), mGeom->getStack(idChamber), sector);
 
   // 0: InFlight 1: Entering 2: Exiting
   int trkStat = 0;
+  double xp, yp, zp, px, py, pz, etot;
```

Comment:
do you need doubles here or would float be enough?

Reply:
Hello @sawenzel, thanks for the suggestion. Now I paid closer attention to this and changed all the occurrences for stored variables to float. I think it is ok if I treat them like that for here to the future. There are many places in the code that uses double, but only as part of the algorithms, I guess that's ok.